### PR TITLE
Create a test case for report gen

### LINKED ISSUE #26: Create tests/test_report_generator.py test_report_gen()

### PR DESCRIPTION
**Pull request by ameliav**
#  #

## 🗣 Description ##
This PR resolves issue https://github.com/cisagov/tpt-reports/issues/26 and creates a test case for the report_gen function in report_generator.py.

## 💭 Motivation and context ##
This will add a test case to an untested function and give more coverage.

## 🧪 Testing ##
All automated tests, including this new test, were successful.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [ ] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

File: tests/test_report_generator.py [hunk 1]
```diff
@@ -13,16 +15,40 @@
 # cisagov Libraries
 from tpt_reports import report_generator
 
+FILE_PATH = "./tests/data"
```
Comment (dav3r) [inline tests/test_report_generator.py]: You should use the full path to the test JSON file, as is done in the other test:
https://github.com/cisagov/tpt-reports/blob/807cbe870fddb0b63ea9c89cf72f5d1723510be4/tests/test_tpt_reports.py#L27

If you need to get the directory that the test file resides in, you can use [`os.path.dirname()`](https://docs.python.org/3/library/os.path.html#os.path.dirname).

File: tests/test_report_generator.py [hunk 2]
```diff
@@ -13,16 +15,40 @@
...
+    """Define a fixture for the payloads DataFrame."""
+    with open(os.path.join(FILE_PATH, "test.json"), encoding="utf-8") as file:
         data = json.load(file)
         return pd.DataFrame.from_dict(data["payloads"])
 
 
-def test_format_table(test_dataframe):
+@pytest.fixture
+def test_dictionary():
+    """Define a fixture for the test json dictionary."""
```
Comment (dav3r) [inline tests/test_report_generator.py]: Use proper capitalization:
```suggestion
    """Define a fixture for the test JSON dictionary."""
```

**Review by dav3r — CHANGES_REQUESTED** [review]
I have a couple of small changes to request.

File: tests/test_report_generator.py (hunk 1 above)
Comment (ameliav) [reply tests/test_report_generator.py]: Resolved in [8c1eb5f](https://github.com/cisagov/tpt-reports/pull/56/commits/8c1eb5fcd8768027c58a5a223491268a233f4cc2)

**Review by schmelz21 — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
👍 👍

**Pull request merged**